app/controllers/case_logs_controller.rb

```diff
@@ -7,7 +7,9 @@ class CaseLogsController < ApplicationController
   before_action :find_resource, except: %i[create index edit]
 
   def index
-    @pagy, @case_logs = pagy(current_user.case_logs)
+    set_session_filters if params[:status].present?
```

Comment:
If filters are just url params do we actually need to store them as sessions at all?

Reply:
I thought if someone visits `/logs<filter_here>` and then goes to `/logs`, we'd still want to keep the filters?

Reply:
Interesting, I'm not sure. I think my expectation would have been that if I press back they're kept but not if I do a fresh page load. @roobottom do you have thoughts?

Reply:
That's the flow yeah. I think my assumption would be that if I've done my thing and then clicked the Logs link again, I'd expect to be starting a new search (or create a new log).

In terms of taking it to the wider group are you comfortable making a decision to be this PR merged and then we revisit if needed in a future ticket?

Reply:
I can make that decision, it works and can always be adjusted upon user feedback 🤷‍♀️